src/pages/DocumentView.vue

```diff
@@ -268,11 +283,11 @@ export default {
     },
     goToPreviousTab() {
       const indexPreviousActiveTab = this.indexActiveTab === 0 ? this.visibleTabs.length - 1 : this.indexActiveTab - 1
-      this.$set(this, 'activeTab', this.visibleTabs[indexPreviousActiveTab].name)
+      this.activeTab = this.visibleTabs[indexPreviousActiveTab].name
```

Comment:
Please avoid to migrate `this.$set` to javascript primitives or it's going to create a lot of conflict when merging the branch to migrate to Vue 3 :grimacing:

Reply:
Ok I'm reverting